### ISSUE TITLE
Text input taking 2-3 sceonds per character in IE 11

### ISSUE THREAD

**Issue by C-Kearney** [codeborne/selenide#772]
## The problem
I am writing a test suite using cucmber to test a web applications UI. We recently finished writing support for IE 11. When I run the tests against IE 11 text input will take 2-3 seconds per character making the tests take up to 4X as long. Any ides on how to make IE 11 run as quickly as our firefox and chrome tests?

## Tell us about your environment
Selenide Version: 4.12.2
IE Version: 11.551.16299.0
Browser Driver Version: 3.13.0.0
OS Version: Windows 10

**Comment by BorisOsipov** [comment]
Did you try win32 version of IE driver?

**Comment by BorisOsipov** [comment]
No I don't. 
But as I know there was huge performance degradation on using win64 driver.

Also you may try to use `Configuration.fastSetValue = true;` It will enable setting values for input fields by js scripts. I've never used it with IE yet, but it works well on some applications.

Also I believe this is not Selenide issue... You will have the same problems with plain Selenium.

**Comment by C-Kearney** [comment]
Yeah I saw a lot of the Selenium threads mentioning this issue and I wondered if anyone working on this project had figured out a faster solution.

Thanks for the advice!

**Comment by asolntsev** [comment]
@C-Kearney yes, `Configuration.fastSetValue = true` should help in your case. Did you try it?
